pkg/audit/audit.go

```diff
@@ -70,3 +71,10 @@ func (a *Audit) Log(name, op, value string) {
 	}
 	a.mutex.Unlock()
 }
+
+func Delete(file string) {
+	check_file, _ := os.Stat(file)
```

Comment:
what happens when file doesn't exist for some reason? its better to check for err and do nothing if error occurred.

Reply:
Done